src/components/Typography/index.js

```diff
@@ -0,0 +1,7 @@
+import { Typographies } from './styles';
+
+export const Typography = ({ children, varient, color }) => {
+  const Element = Typographies[varient];
+
+  return <Element color={color}>{children}</Element>;
+};
```

Comment:
Adicionar uma linha de elemento default, na hora de usar sem passar variant isso vai gerar um erro.

```suggestion
};

Typography.defaultProps = {
variant: 'h1Regular'
}
```